bin/git-forgit

```diff
@@ -1002,6 +1003,98 @@ _forgit_ignore_clean() {
     [[ -d "$FORGIT_GI_REPO_LOCAL" ]] && rm -rf "$FORGIT_GI_REPO_LOCAL"
 }
 
+_forgit_worktree_preview() {
+    local sha
+    # trailing space in  grep to avoid matching worktrees with a common path
+    sha=$(git worktree list | grep "$1 " | awk '{print $2}')
+    # bare git-dir has no history
+    [[ "$sha" == "(bare)" ]] && return
+    _forgit_worktree_preview_git_opts=()
+    _forgit_parse_array _forgit_worktree_preview_git_opts "$FORGIT_WORKTREE_PREVIEW_GIT_OPTS"
+    # the trailing '--' ensures that this works for branches that have a name
+    # that is identical to a file
+    git log "$sha" "${_forgit_worktree_preview_git_opts[@]}" --
+}
+
+_forgit_worktree_jump() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
+    local count tree opts
+    count=$(git worktree list | wc -l)
+
+    [[ $count -eq 1 ]] && return 1
+
+    opts="
+        $FORGIT_FZF_DEFAULT_OPTS
+        +s +m --tiebreak=index
+        --preview=\"$FORGIT worktree_preview {1}\"
+        $FORGIT_WORKTREE_JUMP_FZF_OPTS
+    "
+
+    tree=$(git worktree list | awk '{print $1}' | FZF_DEFAULT_OPTS="$opts" fzf)
+    [[ -z "$tree" ]] && return 1
+    echo "$tree"
+}
+
+_forgit_git_worktree_lock() {
+    _forgit_worktree_lock_git_opts=()
+    _forgit_parse_array _forgit_worktree_lock_git_opts "$FORGIT_WORKTREE_LOCK_GIT_OPTS"
+    git worktree lock "${_forgit_worktree_lock_git_opts[@]}" "$@"
+}
+
+_forgit_worktree_lock() {
+    _forgit_inside_work_tree || _forgit_inside_git_dir || return 1
```

Comment:
I guess I never thought of that. I wanted to add other commands like `git worktree add` as `gwa` but wasn't sure where to include the fzf selecting since it has several optional parts

```sh
git worktree add [-f] [--detach] [--checkout] [-b <new-branch>] <path> [<branch>]
```

Reply:
> I wanted to add other commands like git worktree add as gwa but wasn't sure where to include the fzf selecting since it has several optional parts

Not all git command fit forgits ergonomics, I think the selection you made is sensible. We can always add other commands later on if we come up with something.